Index: src/daemon/Makefile.am
===================================================================
--- src/daemon/Makefile.am
+++ src/daemon/Makefile.am	2007-01-03 09:59:16.000000000 +0100
@@ -23,6 +23,7 @@
 
 notificationdaemon-dbus-glue.h: notificationdaemon.xml
 	dbus-binding-tool --mode=glib-server 				\
+		--prefix=notification_daemon_xfce \
 		$(srcdir)/notificationdaemon.xml > notificationdaemon-dbus-glue.h
 
 INCLUDES =								\
Index: src/daemon/Makefile.in
===================================================================
--- src/daemon/Makefile.in
+++ src/daemon/Makefile.in	2007-01-03 09:59:05.000000000 +0100
@@ -582,6 +582,7 @@
 
 notificationdaemon-dbus-glue.h: notificationdaemon.xml
 	dbus-binding-tool --mode=glib-server 				\
+		--prefix=notification_daemon_xfce \
 		$(srcdir)/notificationdaemon.xml > notificationdaemon-dbus-glue.h
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
Index: src/daemon/daemon.c
===================================================================
--- src/daemon/daemon.c
+++ src/daemon/daemon.c	2007-01-03 09:59:46.000000000 +0100
@@ -1108,7 +1108,7 @@
 	dbus_conn = dbus_g_connection_get_connection(connection);
 
 	dbus_g_object_type_install_info(NOTIFY_TYPE_DAEMON,
-									&dbus_glib__object_info);
+									 &dbus_glib_notification_daemon_xfce_object_info);
 
 	bus_proxy = dbus_g_proxy_new_for_name(connection,
 										  "org.freedesktop.DBus",
